src/main.rs

```diff
@@ -26,13 +26,13 @@ fn main() {
     println!("nf3 {:?}", cb.to_string());
 }
 
-//min function
-#[rustfmt::skip]
-fn min<T: Ord>(a: T, b: T) -> T {
-    if a < b { a } else { b }
-}
-//max function
-#[rustfmt::skip]
-fn max<T: Ord>(a: T, b: T) -> T {
-    if a > b { a } else { b }
-}
+// //min function
+// #[rustfmt::skip]
+// fn min<T: Ord>(a: T, b: T) -> T {
+//     if a < b { a } else { b }
+// }
+// //max function
+// #[rustfmt::skip]
+// fn max<T: Ord>(a: T, b: T) -> T {
+//     if a > b { a } else { b }
+// }
```

Comment:
We can use #[rustfmt::skip] to skip formatting, I wonder if we can use something simialr to skip linting with clippy.